### PR TITLE
Run analysis on Hyper

### PR DESCRIPTION
**Pull request by olafveerman**
This allows the analysis process to be run on Hyper, instead of a Docker container. Will mostly useful for running the tool in a hosted environment.

To test it, you'd need to:

1. create a Hyper account and add the keys to the config or as env vars
2. have the API, DB and Minio run on a machine that can be accessed from the outside

### CODE REVIEW

File: app/config/staging.js [hunk 1]
```diff
@@ -15,5 +15,13 @@ module.exports = {
     secretKey: 'miniostorageengine',
     bucket: 'rra',
     region: 'us-east-1'
+  },
+  analysisProcess: {
```
Comment (danielfdsilva) [inline app/config/staging.js]: Missing `hyperSecret`?

File: app/config/production.js [hunk 2]
```diff
@@ -26,6 +26,8 @@ module.exports = {
     region: null
   },
   analysisProcess: {
+    service: null,
+    hyperAccess: null,
```
Comment (danielfdsilva) [inline app/config/production.js]: Missing `hyperSecret`?

File: app/routes/scenarios--gen-results.js [hunk 3]
```diff
@@ -119,13 +123,29 @@ function spawnAnalysisProcess (projId, scId, opId) {
...
+        '-e', `HYPER_ACCESS=${config.analysisProcess.hyperAccess}`,
+        '-e', `HYPER_SECRET=${config.analysisProcess.hyperSecret}`
+      );
+      break;
+    default:
+      console.log(`${service} is not a valid option. The analysis should be run on 'docker' or 'hyper'. Check your config file or env variables.`);
+  }
+
+  // Append the name of the image last
+  let args = baseOptions.concat(config.analysisProcess.container);
```
Comment (danielfdsilva) [inline app/routes/scenarios--gen-results.js]: Maybe use always the same approach when adding elements to the array. Call it `args` from the start and always use `push`, seems more consistent.

File: app/config/staging.js (hunk 1 above)
Comment (olafveerman) [inline app/config/staging.js]: In a staging or production environment, it should really be in an environment variable.
By keeping it out of `staging.js` and `production.js`, it's harder for people to add the secret by mistake.

Comment (danielfdsilva) [inline app/config/staging.js]: I usually add them in so there's a base to copy for local.
Should we add them in with a comment in front, or remove the others as well?

**Pull request merged**